velox/functions/sparksql/Register.cpp

```diff
@@ -203,6 +203,8 @@ void registerFunctions(const std::string& prefix) {
       {prefix + "make_date"});
 
   registerFunction<LastDayFunction, Date, Date>({prefix + "last_day"});
+  registerFunction<AddMonthsFunction, Date, Date, int32_t>(
```

Comment:
> override def inputTypes: Seq[AbstractDataType] = Seq(DateType, IntegerType)

Looks the input type in Spark can also be integer type, can we support that at the same time?

Reply:
the first `Date` is the return type, and the input type is `Date, int32_t` which matches the spark signature.